verification/tests/sourcify.rs

```diff
@@ -0,0 +1,42 @@
+use actix_web::{
+    test::{self, TestRequest},
+    App,
+};
+use serde_json::json;
+use verification::{routes, Config};
+
+#[actix_rt::test]
+async fn should_return_200() {
+    let config = Config::parse().expect("Failed to parse config");
+    let mut app = test::init_service(
+        App::new().configure(|service_config| routes::config(service_config, config)),
+    )
+    .await;
+
+    let metadata = include_str!("contracts/storage/metadata.json");
+    let source = include_str!("contracts/storage/source.sol");
+    let request_body = json!({
+        // relies on the fact that the kovan testnet has this contract
+        // https://kovan.etherscan.io/address/0xfbe36e5cad207d5fdee40e6568bb276a351f6713
+        "address": "0xFBe36e5cAD207d5fDee40E6568bb276a351f6713",
+        "chain": "42",
+        "files": {
+            "source.sol": source,
+            "metadata.json": metadata,
+        }
+    });
+
+    let resp = TestRequest::get()
```

Comment:
what's the difference between `TestRequest` and `reqwest`? 

Also, what if the tests are runned on the machine with no internet connection (it can be some CI/CD runner in the datacenter), or sourcify is down? 

Are we testing, that current sourcify api will return the correct response (i.e. it didn't change the api), or are we testing, that we're calling the sourcify api correctly? If it's the second, may be we want to mock the sourcify api then? And I don't think, that we're really need the first kind of tests, but if we are, I think it'll be better to do as a separate test

Reply:
But if we decide to change the test, I suggest doing it in another PR

Reply:
+ we didn't start server on some port, TestRequest just calls service of app.

+ We are testing whole endpoint, checking that it works correctly. I decided not to write mock for soucify because I don't think we need it right now. When tests will fail due to internet connection, then we will fix it.

+ Yes, we can change the test in another PR

Reply:
> And I don't think, that we're really need the first kind of tests, but if we are, I think it'll be better to do as a separate test

Why?

Reply:
> Why?

I can trust sourcify not to change their api without backward compability, and I think we don't need these tests _at least right now_

Reply:
> we didn't start server on some port, TestRequest just calls service of app.

Ah, thanks, that's convinient

> I decided not to write mock for soucify because I don't think we need it right now

Fair enough